mmv1/products/dlp/api.yaml

```diff
@@ -1328,4 +1330,267 @@ objects:
+                        - Redact a field if the date of birth field is greater than 85.
+                      properties:
+                        - !ruby/object:Api::Type::NestedObject
+                          name: expressions
+                          description: An expression.
+                          properties:
+                            - !ruby/object:Api::Type::Enum
+                              name: logicalOperator
+                              description: The operator to apply to the result of conditions. Default and currently only supported value is AND
+                              default_value: :AND
+                              values:
+                                - :AND
+                            - !ruby/object:Api::Type::NestedObject
+                              name: conditions
+                              description: Conditions to apply to the expression.
+                              properties:
+                                - !ruby/object:Api::Type::Array
+                                  name: conditions
+                                  description: A collection of conditions.
+                                  item_type: !ruby/object:Api::Type::NestedObject
+                                    properties:
+                                      - !ruby/object:Api::Type::NestedObject
+                                        name: field
+                                        description: Field within the record this condition is evaluated against.
+                                        required: true
+                                        properties:
+                                          - !ruby/object:Api::Type::String
+                                            name: name
+                                            description: Name describing the field.
+                                      - !ruby/object:Api::Type::Enum
+                                        name: operator
+                                        description: Operator used to compare the field or infoType to the value.
+                                        required: true
+                                        values:
+                                          - :EQUAL_TO
+                                          - :NOT_EQUAL_TO
+                                          - :GREATER_THAN
+                                          - :LESS_THAN
+                                          - :GREATER_THAN_OR_EQUALS
+                                          - :LESS_THAN_OR_EQUALS
+                                          - :EXISTS
+                                      - !ruby/object:Api::Type::NestedObject
+                                        name: value
+                                        description: Value to compare against. [Mandatory, except for EXISTS tests.]
```

Comment:
it looks like the nested fields here are all exactly_one_of, but if I recall correctly, that doesn't work nested inside an array? I assume that's why it's not marked here. Could be worth adding something to the documentation instead?

Reply:
Yeah that's correct about arrays not allowing exactly_one_of checks 😞 , but updating the documentation sounds like a good alternative

Reply:
Done in [ecf83d6](https://github.com/GoogleCloudPlatform/magic-modules/pull/6872/commits/ecf83d6134db5a328522cc87faa8e623af763377)